tools/data-handler/src/containers/template.ts

```diff
@@ -172,8 +172,10 @@ export class Template extends CardContainer {
             for (const directory of directories) {
                 const dirPath = join(directory.path, directory.name);
                 if (pathExists(resolve(templatePath))) {
-                    templatePath = join(dirPath, 'templates', templateName);
-                    break;
+                    if (pathExists(join(dirPath, 'templates', templateName))) {
```

Comment:
Did you mean to remove the first if-statement?

Reply:
No, it is now checking if the path exists and then use the path. 
--> It is template from a module
If this if fails, then it cannot be module template, but a local one.

Reply:
The previous `if` is probably not needed at all (row 174).

Reply:
I think I will re-write this whole method; it is a mess.

Reply:
I think it is now a lot clearer.